content/docs/start/data-pipelines.md

```diff
@@ -38,6 +38,8 @@ Now let's install the requirements:
 > first.
 
 ```dvc
+$ python3 -m venv .env
+$ source .env/bin/activate
 $ pip install -r src/requirements.txt
 ```
```

Comment:
If this stays, it should be part of the md block quote above, please.

But should it stay? I'm not sure. We purposefully don't include this in our docs any more (we try to avoid having to maintain docs/examples of non-DVC tools). We assume people will know some basics. THAT SAID, maybe in this case it's a good idea to keep it, as this is the most introductory of our docs.

Reply:
Also,

> source .env/bin/activate

On Windows it's

`source .env/Scripts/activate`

I know 😠 but that's another possible reason to avoid this.

Reply:
Hmm, got it. One cool thing spaCy has is [this quick section to generate the command](https://spacy.io/usage#quickstart). But it makes more sense there because it's the first page of the Get Started guide there. Here the page in question is the "third one" of the guide, I don't know if the pages should follow a sequence or not.

About assuming people will know some basics, maybe it would be good to add [here](https://dvc.org/doc/user-guide/contributing/docs) a little more about what you guys assume about the audience of the docs?

Reply:
@dmesquita I've seen a few projects do a widget likes this for the installation page ... we can do something similar for the dvc.org/doc/install - that would be great. Please, feel free to create a ticket.